README.md

```diff
@@ -15,7 +15,7 @@ A bro will lint and prettify your code
 $ yarn add @codementor/bro --dev
 ```
 
-To let your editor lint plugin works, you'll need to copy the `.eslintrc` from this package to your project root.
+To let your editor lint plugin works, you'll need to copy the `eslintrc.json` from this package to your project root and rename to `.eslintrc.json`.
 ```
 $ cp node_modules/@codementor/bro/eslintrc.json ./.eslintrc
```

Comment:
How about changing the script to `cp node_modules/@codementor/bro/eslintrc.json ./`?